.gitmodules

```diff
@@ -29,7 +29,7 @@
 	url = git://github.com/mono/NUnitLite.git
 [submodule "external/buildscripts"]
 	path = external/buildscripts
-	url = git@github.com:Unity-Technologies/monobuildtools.git
+	url = https://github.com/Unity-Technologies/monobuildtools.git
```

Comment:
I think that you will need to rebase on unity-2017-02, as @joncham has made the external/buildscripts directory no longer a submodule.